### ISSUE TITLE
Build IOS Building demo: double-conversion | Running 1 of 1 custom shell scripts

### ISSUE THREAD

**Issue by geekwill** [facebook/react-native#18982]
## Environment
react-native-cli: 2.0.1
react-native: 0.55.3
OS: macOS High Sierra 10.13.4
Node: 8.11.1
Yarn: Not Found
npm: 5.6.0
Watchman: 4.9.0
Xcode: Xcode 9.3 Build version 9E145
Android Studio: Not Found

## Steps to Reproduce
This is a whole new project using react-native init.
1： Using the react-native run-ios startup project will stay on the way (with screenshots below), and no further execution.
2： Open the project in Xcode, and start the emulator to stay in the "Building demo: double-conversion | Running 1 of 1 custom shell scripts", and the cut is no longer executed without any error.

## Expected Behavior
The program is executed normally and the simulator can be started.

## Actual Behavior
![wechatimg1](https://user-images.githubusercontent.com/23651539/39103583-25835afc-46de-11e8-9adf-1635ffa09baf.jpeg)
![1524449285377](https://user-images.githubusercontent.com/23651539/39103603-3e321fc0-46de-11e8-9105-7d42fab51880.jpg)

I'm not sure that my question is the same as #15469  , and the solution I gave in  #15469  doesn't seem to solve my problem.

**Comment by geekwill** [comment]
@react-native-bot Thanks for your reply. I have executed the react-native info according to your requirements, and I have edited the Environment information of issue.

**Comment by timwangdev** [comment]
Here is the custom shell script that blocks you:
https://github.com/facebook/react-native/blob/master/scripts/ios-install-third-party.sh
Try to set up a proxy for github and s3 urls.

**Comment by geekwill** [comment]
@timwangdev Thank you for your reply. I did find out that it was caused by the network. What puzzles me is that my development environment is proxies. Fortunately I saw @pampang answer in #14423  and solved my problem.
I manually downloaded the four `.tar.gz` files required for `node_modules/react-native/ios-install-third-party.sh` to local, copied to `~/.rncache/` and re-executed `react-native run-ios` command to achieve my desired effect, the program starts normally.
